src/CloudFtpBridge.Core/Extensions/Microsoft.Extensions.DependencyInjection/IServiceCollectionExtensions.cs

```diff
@@ -14,6 +14,7 @@ public static IServiceCollection AddCloudFtpBridge(this IServiceCollection servi
             services
                 .Configure<CoreOptions>(configuration.GetSection("CloudFtpBridge"))
                 .AddSingleton<FileSystemActivator>()
+                .AddSingleton<FTPSystemActivator>()

```

Comment:
This can be removed. See comments on FTPSystemActivator.

Reply:
This has been resolved in the latest commit to AddNewFTPOption branch.